src/async_client.cpp

```diff
@@ -32,7 +32,7 @@ namespace mqtt {
 
 /////////////////////////////////////////////////////////////////////////////
 
-async_client::async_client(const std::string& serverURI, const std::string& clientId)
```

Comment:
Persona opinion: the removal of `std` namespace could be done in another commit. This eases the review process.